### DISCUSSION TITLE
TOC and links to other formats not visible in mobile view?

### DISCUSSION THREAD

**Discussion by pipme** [quarto-dev/quarto-cli#5896] [Q&A] [answered]
### Description

In mobile view, I can't find the links to other formats like revealjs, pdf etc, neither table of contents. Is it a bug or intended behaviours?

For example, https://pipme.github.io/posts/2023-06-11-Quarto-tips/index-gist.html in the desktop:
![image](https://github.com/quarto-dev/quarto-cli/assets/30783979/35dc596b-78e5-4506-a80b-b0addf9cd9dc)

While in mobile Chrome:
![IMG_6381807EEFD0-1](https://github.com/quarto-dev/quarto-cli/assets/30783979/38e8a46d-29e2-45c7-bd54-5c23ab7cb3ed)

**Comment by mcanouil** [comment]
@dragonstyle Should I convert this to an issue or is it intended?

**Reply by dragonstyle** [reply]
It is as designed right now, but I think it is a real problem that ideally we would address. An issue would be appreciated- probably an enhancement... Ty!!!

**Comment by mcanouil** [comment, marked as answer]
Closing this in favour of #5961
